packages/rest-api-client/src/client/types/app/notification.ts

```diff
@@ -0,0 +1,9 @@
+export type GeneralNotificationsForResponse = {
+  entity: Record<"type" | "code", string>;
```

Comment:
I think it is good that the values of `type` are literal types, not a string.
You can use the `Entity` type and connect it with the `FIELD_ENTITY` type like the below.
https://github.com/kintone/js-sdk/blob/e0d502e64158fad2a50fd1b6a1a946fbeaa3a2ee/packages/rest-api-client/src/client/types/app/right.ts#L70

```suggestion
  entity:
    | Entity
    | {
        type: "FIELD_ENTITY";
        code: string;
      };
```

NOTE: You cannot apply this suggestion directly because it needs importing the `Entity` type.